src/sentry/api/bases/organization_events.py

```diff
@@ -1,18 +1,32 @@
 from __future__ import absolute_import
 
 import six
+from datetime import timedelta
 from rest_framework.exceptions import PermissionDenied
 from rest_framework.exceptions import ParseError
 
 
 from sentry import features
 from sentry_relay.consts import SPAN_STATUS_CODE_TO_NAME
 from sentry.api.bases import OrganizationEndpoint, OrganizationEventsError
-from sentry.api.event_search import get_filter, InvalidSearchQuery, get_json_meta_type
+from sentry.api.event_search import (
+    get_filter,
+    InvalidSearchQuery,
+    get_json_meta_type,
+    get_function_alias,
+)
+from sentry.api.serializers.snuba import SnubaTSResultSerializer
 from sentry.models.project import Project
 from sentry.models.group import Group
 from sentry.snuba.discover import ReferenceEvent
-from sentry.utils.compat import map
+from sentry.utils.compat import map, zip
+from sentry.utils.dates import parse_stats_period
+
+
+# Maximum number of results we are willing to fetch.
+# Clients should adapt the interval width based on their
+# display width.
+MAX_POINTS = 4500
```

Comment:
We have a few copies of this value. Should we consolidate them into either `sentry.constants`, `sentry.snuba.discover` or `sentry.utils.snuba`?